javav2/example_code/dynamodb/src/main/java/com/example/dynamodb/ScenarioPartiQLBatch.java

```diff
@@ -32,380 +34,425 @@
  * https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
- *
+ * <p>
  * This Java example performs the following tasks:
- *
+ * <p>
  * 1. Creates the Amazon DynamoDB movie table with a partition and sort key.
  * 2. Puts new records into the table using a BatchExecuteStatement
  * 3. Updates items using a BatchExecuteStatement.
  * 4. Deletes items by using a BatchExecuteStatement.
  * 5. Deletes the table.
- *
+ * <p>
  * To see another code example with more options using PartiQL, see the
  * ScenarioPartiQ code example.
  */
 
 // snippet-start:[dynamodb.java2.scenario.partiql.batch.main]
 public class ScenarioPartiQLBatch {
-        public static void main(String[] args) throws IOException {
-                String tableName = "MoviesPartiQBatch";
-                Region region = Region.US_EAST_1;
-                DynamoDbClient ddb = DynamoDbClient.builder()
-                                .region(region)
-                                .build();
-
-                System.out.println("******* Creating an Amazon DynamoDB table named " + tableName
-                                + " with a key named year and a sort key named title.");
-                createTable(ddb, tableName);
-
-                System.out.println("******* Adding multiple records into the " + tableName
-                                + " table using a batch command.");
-                putRecordBatch(ddb);
-
-                System.out.println("******* Updating multiple records using a batch command.");
-                updateTableItemBatch(ddb);
-
-                System.out.println("******* Deleting multiple records using a batch command.");
-                deleteItemBatch(ddb);
-
-                System.out.println("******* Deleting the Amazon DynamoDB table.");
-                deleteDynamoDBTable(ddb, tableName);
-                ddb.close();
+    public static void main(String[] args) throws IOException {
+        String tableName = "MoviesPartiQBatch";
+        Region region = Region.US_EAST_1;
+        DynamoDbClient ddb = DynamoDbClient.builder()
+            .region(region)
+            .build();
+
+        System.out.println("******* Creating an Amazon DynamoDB table named " + tableName
+            + " with a key named year and a sort key named title.");
+        createTable(ddb, tableName);
+
+        System.out.println("******* Adding multiple records into the " + tableName
+            + " table using a batch command.");
+        putRecordBatch(ddb);
+
+        // Update multiple movies by using the BatchExecute statement.
+        String title1 = "Star Wars";
+        int year1 = 1977;
+        String title2 = "Wizard of Oz";
+        int year2 = 1939;
+
+        System.out.println("Updating two movies with producer information: " + title1 + " and " + title2 + ".");
+        getBatch(ddb, tableName, title1, title2, year1, year2);
+
+        System.out.println("******* Updating multiple records using a batch command.");

```

Comment:
Don't use asterisks like this unless you provide the option to disable them. It's very annoying to a screen reader user.